src/main/java/www/purple/mixxy/views/layout/footer.ftl.html

```diff
@@ -5,7 +5,7 @@
         <li><a href="/privacy">Privacy Policy</a></li>
         <li><a href="/about">About</a></li>
         <li><a href="/terms">Terms of Use</a></li>
+        <li id="copy">&copy; 2016 www-purple</li>
```

Comment:
It's actually "www.purple" (note the dot), the GitHub org is just called "www-purple" (note the dash) because it doesn't support `.` in the name.